### ISSUE TITLE
Failure during `pulumi refresh` for `pulumi-random` when ingesting HEAD of pulumi-terraform

### ISSUE THREAD

**Issue by ellismg** [pulumi/pulumi-terraform#281]
Trying to update the `pulumi-terraform` reference in `pulumi-random` and running into this problem during the refresh operation:

```
[ /pulumi-random/examples/simple ] Diagnostics:
[ /pulumi-random/examples/simple ]   random:index:RandomInteger: integer
[ /pulumi-random/examples/simple ]     error: Plan apply failed: importing 932: Invalid import usage: expecting {result},{min},{max} or {result},{min},{max},{seed}
```

https://travis-ci.com/pulumi/pulumi-random/builds/91115873#L1123 shows the failure.

**Comment by lukehoban** [comment]
Added a longer note on https://github.com/pulumi/pulumi-terraform/pull/282#issuecomment-438322428, but I think this raises the issue that it's not necessarily safe to call `Importer` in `pulumi refresh` scenarios.

**Comment by lukehoban** [comment]
We already fixed this - but by reverting #275.  This now tracks redoing #275 and avoiding this issue.
